itests/hive-blobstore/pom.xml

```diff
@@ -55,33 +55,33 @@
     </dependency>
     <dependency>
       <groupId>org.apache.hive</groupId>
-      <artifactId>hive-metastore</artifactId>
+      <artifactId>hive-exec</artifactId>
       <version>${project.version}</version>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.hive</groupId>
       <artifactId>hive-metastore</artifactId>
       <version>${project.version}</version>
-      <classifier>tests</classifier>
       <scope>test</scope>
     </dependency>
     <dependency>
       <groupId>org.apache.hive</groupId>
-      <artifactId>hive-it-unit</artifactId>
+      <artifactId>hive-metastore</artifactId>
```

Comment:
hmm why we need two `hive-metastore` dependencies (one with tests classifier)? I don't see this in the original patch.

Reply:
Oh, I don't change it actually. 

This diff looks like I add new `hive-metastore` dependency but in the original pom.xml, it already includes two `hive-metastore`, one is without classifier and one is with `tests` classifier.

Just the way git showing diff confusing readers.

Reply:
Oh cool. Thanks.